test/integration/integration_test.go

```diff
@@ -214,34 +243,52 @@ func (i *integrationTest) exerciseCleanBuild(tag string, verifyCallback bool, im
 		t.Fatalf("Sti build did not invoke callback with valid json message")
 	}
 
-	i.checkForImage(tag)
-	containerID := i.createContainer(tag)
-	defer i.removeContainer(containerID)
-	i.checkBasicBuildState(containerID, resp.WorkingDir)
+	if expectImageName {
+		if resp.ImageID == "" {
+			t.Fatalf("Sti build did not receive an imageID in response")
```

Comment:
I figured it'd still be sti since we're talking about the command `sti build`.  I'll update these lines and any other instances of "sti" to be "s2i"